README.md

```diff
@@ -283,37 +287,41 @@ For the remainder of the background worker documentation, it is assumed that you
 
 ### Terminology
 
-<dl>
-<dt>Resque</dt>
-<dd>Resque is a <a href="https://en.wikipedia.org/wiki/Message_queue">message queue</a> that is used by Sufia to manage long-running or slow processes. (Sufia builds its jobs using Rails's ActiveJob framework, so you are free to use another queuing system, e.g., DelayedJob or Sidekiq.)</dd>
+##### Resque
+
```

Comment:
I consider this ad-hoc replacement for definition list at the correct "h".  If we had enough terms to break into sections, section headings would be h4.